diagnostic.md

```diff
@@ -31,25 +38,26 @@ the `form` DOM node. (Assume the list item has a class of "listerItem").
 Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
-<!-- your answer starts here -->
+Setters_
+$('#Barack Obama').val('Donald Trump') would change the string value of the ID Barack Obama to Donald Trump
+$('#America').val('China') would change the string value of the id America  America to China.
 
+Getters
+$('#Barack Obama').val() would get 'Barack Obama'
+$('#Donald Trump').val() would get "Donald Trum'"
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+The DOM(DOCUMENT OBJECT MODEL) acts as the intermediary between clinet and server. It allows developers to see changes made in their code while. It pulls a given document,(independent of language), parses it, and creates an object model of the document in Node Tree Form
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.  A web browser implements this interface and renders it for you to view and interact with.